src/pseudopeople/noise_entities.py

```diff
@@ -77,15 +77,13 @@ class __NoiseTypes(NamedTuple):
     #         Keys.TOKEN_PROBABILITY: 0.1,
     #     },
     # )
-    # make_ocr_errors: ColumnNoiseType = ColumnNoiseType(
-    #     "make_ocr_errors",
-    #     noise_functions.make_ocr_errors,
-    #     probability=None,
-    #     additional_parameters={
-    #         Keys.CELL_PROBABILITY: 0.01,
-    #         Keys.TOKEN_PROBABILITY: 0.1,
-    #     },
-    # )
+    ocr: ColumnNoiseType = ColumnNoiseType(
```

Comment:
change this to `make_ocr_errors`

Reply:
This was the cause of a merge conflict and has been updated.